locations.md

```diff
@@ -0,0 +1,41 @@
+# Locations of files
+This is a list of files downloaded by the extension, and where they're installed. If the extension fails to download something, then you can manually install these files to the specified locations and still be able to use the extension.
+
+> Note `~` is used to denote the user's home directory, and anything enclosed by `<...>` is a variable you should replace
+
+## SDK
+Installed at `~/.pico-sdk/sdk/<sdk_version>/`, where `sdk_version` is a release (eg 1.5.1 or 2.0.0). Can be manually installed using git
+```
+git -c advice.detachedHead=false clone --branch <sdk_version> https://github.com/raspberrypi/pico-sdk.git "~/.pico-sdk/sdk/<sdk_version>"
+cd "~/.pico-sdk/sdk/<sdk_version>"
+git submodule update --init
```

Comment:
instead of a dedicated step here, perhaps add `--recursive` to the clone command above?

Reply:
That recursively clones all the submodules of the submodules, which aren’t actually needed - we only need the top-level of the submodules, hence the separate steps. I don’t think this is an issue anymore for the submodules in SDK 2.0.0, but in SDK 1.5.1 tinyusb has a bunch of submodules that we don’t want to clone

Reply:
If you also add `--depth 1` it reduces the overall clone size (top level and submodules).